src/serde/value/owned/de.rs

```diff
@@ -842,7 +752,11 @@ impl<'de> VariantAccess<'de> for VariantRefDeserializer<'de> {
                 if v.is_empty() {
                     visitor.visit_unit()
                 } else {
-                     visit_array_ref(v, visitor)
+                    // This is somewhat nasty, but it's worth it since we can otherwise
+                    // avoid the clode by using into_iter instead of iter.
```

Comment:
`clode` -> `clone` otherwise `clode` is a good name for a project!

Reply:
outdated comment, I'll yank it completely (we're no longer cloning)

Reply:
Even better - whack the comment entirely!